app/src/main/res/layout/issue_list_elem.xml

```diff
@@ -32,13 +32,13 @@
         android:layout_height="wrap_content"
         android:layout_alignParentLeft="true"
         android:layout_below="@+id/project_shortcut"
-        android:ellipsize="none"
+        android:ellipsize="start"
+        android:scrollHorizontally="true"
```

Comment:
Sorry Marcello for such a delay, I took holidays and there was an easter also.

It looks ok, but I don't know what this does
```
android:scrollHorizontally="true"
``` 
It doesn't change anything (at least on my phone) even if I have very long issue id (like 6 digits).